### PR TITLE
Youtube Page => v0.0.0.1 con de Tuti.

### PR DESCRIPTION
**Pull request by AngelVDev**

### CODE REVIEW

**Review by davidcoachdev — APPROVED** [review]
Ok

File: app/youtube/components/Home/HomeYoutube.tsx [hunk 1]
```diff
@@ -9,47 +10,104 @@ const HomeYoutube = () => {
   const channelBanner = items[0].brandingSettings?.image.bannerExternalUrl;
   return (
     <main>
-      <header className={styles.youtube__navBarSection}>
-        <input type="text" placeholder="Search" />
-        <button type="submit">Lupa</button>
+      <header className={styles.youtube__searchbarSection}>
```
Comment (Alan-TheGentleman) [inline app/youtube/components/Home/HomeYoutube.tsx:13]: should refactor the code in a way that we have multiple logic sections. 
Here for example we could create a new component that separates the logic of searching through YouTube

File: app/youtube/components/Home/HomeYoutube.tsx [hunk 2]
```diff
@@ -9,47 +10,104 @@ const HomeYoutube = () => {
...
+          aria-label="Buscar"
+          role="combobox"
+          aria-haspopup="false"
+          aria-autocomplete="list"
+          dir="ltr"
+        />
+        <button className={styles.youtubeInputButton} type="submit">
+          <SearchIcon />
+        </button>
         {/* call the "chrome.identity.getProfileUserInfo" function to retrieve the user's basic profile information. Note that the will need to grant your extension permission to access their profile information. To request this permission, you can call the "chrome.identity.getAuthToken" function, which will prompt the user to sign in to their Google account and grant your extension access to their basic profile information.  */}
```
Comment (Alan-TheGentleman) [inline app/youtube/components/Home/HomeYoutube.tsx:33]: Should remove comments before asking for a pull request

File: app/youtube/components/Home/HomeYoutube.tsx [hunk 3]
```diff
@@ -9,47 +10,104 @@ const HomeYoutube = () => {
...
+        className={styles.youtube__banner}
+      ></div>
+      <section>
+        <section className={styles.youtubeChannelDetails}>
+          <img
+            className={styles.youtubeChannelPic}
+            draggable="false"
+            alt="GentlemanProgramming Channel Pic"
+            width="128"
+            src={
```
Comment (Alan-TheGentleman) [inline app/youtube/components/Home/HomeYoutube.tsx:49]: links like this, that are too long, could be defined in a variable

**Pull request merged**